PublicMapping/districtbuilder
# Lock districts for editing

## Overview
Lock districts for editing

### Demo
![db_lock_districts](https://user-images.githubusercontent.com/2926237/89208764-fde3f680-d58a-11ea-85ee-14b50ee4c20f.gif)

## Notes
This copies over the sprites from the prototype for the dotted pattern overlay for locked districts.

This deviates from the prototype in that selecting a partially locked geometry will not select anything instead of selecting only the unlocked sub-geounits. Making it work like the prototype would be much more involved since we'd need to query for multiple geolevel layers to get all the constituent features to enable selecting sub-geounits of partially locked geometries. 

## Testing Instructions
* Lock and unlock a district with some geometries assigned to it to check the dotted overlay is shown/hidden correctly
* Make sure white lock is shown for a row when mousing over and then a filled in lock is shown when a district is locked
* Test that geometries inside of a locked district cannot be selected with any selection tool
* Test that partially locked geometries cannot be selected (eg. select county with locked blockgroups/tracts)

Closes #112
